### ISSUE TITLE
[minikube] Storage Operator does not provision persistent volumes

### ISSUE THREAD

**Issue by philross** [arangodb/kube-arangodb#563]
K8s environment:
`minikube`

Version:
`kube-arangodb:1.0.2`

Hello,

I followed this tutorial: https://www.arangodb.com/docs/stable/tutorials-kubernetes.html
However, after executing all steps on my minikube, my database pod is stuck in pending status:
```
kubectl get pods
NAME                                                 READY   STATUS    RESTARTS   AGE

arango-arango-deployment-operator-65d878fc69-gmhkj   1/1     Running   0          5m43s

arango-arango-deployment-operator-65d878fc69-pz7fn   1/1     Running   0          5m43s

arango-arango-storage-operator-6fb4c57f87-87cqq      1/1     Running   0          5m36s

arango-arango-storage-operator-6fb4c57f87-vpqmx      1/1     Running   0          5m36s

single-server-sngl-gncez2gh-d8b160                   0/1     Pending   0          2m23s

```

I think it is stuck in pending since no persistent volumes are created, even though I am running the storage operator:
```
kubectl get persistentvolumes
No resources found.
```

**Comment by soderluk** [comment]
Same issue here. It used to work, but now that I upgraded minikube to the latest (v1.10.1) it started failing.

**Comment by soderluk** [comment]
@philross: After some more in-depth investigations we came to a conclusion that it's the latest kubernetes version that breaks this, e.g. 1.18.x
To fix this, you should use the latest 1.17. version of kubernetes in minikube. Start the minikube instance with `minikube start --kubernetes-version=v1.17.6`.
Hope this helps!

**Comment by philross** [comment]
@soderluk Thank you! Setting the kubernetes version to v1.17.6 worked for me.
